src/index.js

```diff
@@ -112,6 +112,18 @@ const ruleFunction = (properties, options, context = {}) => (root, result) => {
   const autoFixFuncNormalized = getAutoFixFunc(autoFixFunc)
   const reKeywords = ignoreKeywords ? {} : null
   const reValues = ignoreValues ? {} : null
+  let cssLoaderValues = []
+
+  if (ignoreVariables) {
+    root.walkAtRules('value', (rule) => {
+      const { params } = rule
+      const name = params.split(':')[0].trim()
+
+      cssLoaderValues.push(name)
+    })
+
+    cssLoaderValues = new RegExp(`^-?(:?${cssLoaderValues.join('|')})$`)
+  }
```

Comment:
Will [`@import`'s](https://www.npmjs.com/package/css-loader#import) of other CSS files also import [`@value`](https://www.npmjs.com/package/css-loader#values) rules?

> You can use @value to specific values to be reused throughout a document.